Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -7136,14 +7143,26 @@ on revIDESaveStack pStackID
    
    lock messages
    
+   local tStackFilename
+   put the effective filename of stack tStackName into tStackFilename
+   
    local tStackFileVersion
    
-   if the cPreserveStackVersion of stack "revPreferences" is true and the cREVGeneral["stackfileversion"] of stack tStackName is not empty then
-      try
-         put the cREVGeneral["stackfileversion"] of stack tStackName into tStackFileVersion
```

Comment:
@livecodeali I searched all stacks and could only find these two spots in the ide library with mention of `cREVGeneral["stackfileversion"]` so I think I've successfully killed it.